source/Modules/storageManager.js

```diff
@@ -141,3 +142,29 @@ export const getBatchTransactions = (cb) => {
     cb(state?.batchTransactions || defaultValue);
   });
 };
+
+export const isAnyWalletConnectedToUrl = (url, walletIds, cb) => {
+  let anyConnection = false;
+  walletIds.forEach((id) => {
+    getApps(id, (_apps = {}) => {
+      if (_apps[url]?.status === CONNECTION_STATUS.accepted) {
```

Comment:
We could use the `getApp` method instead to avoid the `_apps[url]`

Reply:
💯